src/osemgrep/core/Metrics_.ml

```diff
@@ -254,6 +256,11 @@ let string_of_metrics () = Semgrep_metrics_j.string_of_payload g.payload
 (*****************************************************************************)
 (* add_xxx wrappers *)
 (*****************************************************************************)
+let add_is_diff_scan (is_diff_scan : bool) =
```

Comment:
I would remove those functions in osemgrep. I know pysemgrep is doing it, but I think it's stupid,
It just forces us to add one function for every field ... just access the field directly
at the call site.
I would use a function if we do something complicated with the argument, for example add_errors does the preprocessing on the errors to fit in the metrics data structure, but here we don't do anything.